components/aws/sagemaker/batch_transform/src/batch_transform.py

```diff
@@ -61,6 +62,12 @@ def main(argv=None):
   client = _utils.get_sagemaker_client(args.region, args.endpoint_url)
   logging.info('Submitting Batch Transformation request to SageMaker...')
   batch_job_name = _utils.create_transform_job(client, vars(args))
+
+  def signal_term_handler(signalNumber, frame):
+    logging.info(f"Stopping Transform Job: {batch_job_name}")
```

Comment:
Do we care at all about signal number? Or just any `SIGTERM` is fine?

Reply:
nit: Formatting of this message not inline with other messages. Suggestion: `Batch Transform job stopping...`.

Reply:
> we care at all about signal number? 
This method will only be invoked for if process receives SIGTERM because it is only registered for that signal in line 68. We dont care about the type of signal in context of this method but we need to stick to the signature

> just any SIGTERM is fine?
SIGTERM is a type of kill signal. 

>  Formatting of this message
looking for suggestions here. Dont want to use the word stopping. how about changing the log line to `Transform job: request submitted to Stop` and print it after the request is submitted, wdyt?

Reply:
Sounds good :)